tests/unit/challenges/test_views.py

```diff
@@ -5879,24 +5909,43 @@ def test_create_challenge_using_github_success(self):
             resp.content = self.test_zip_file.read()
             resp.status_code = 200
             m.return_value = resp
-            response = self.client.post(
-                self.url,
-                {
-                    "GITHUB_REPOSITORY": "https://github.com/yourusername/repository",
-                    "zip_configuration": self.input_zip_file,
-                },
-                format="multipart",
-            )
-            expected = {
-                "Success": "Challenge Challenge Title has been created successfully and sent for review to EvalAI Admin."
+
+            yaml_file_data = {
+                "worker_image_url": "https://example.com/worker_image.png",
```

Comment:
This is a sample docker image. Actually, my bad. Let's remove the tests for now.